coderd/searchquery/search.go

```diff
@@ -95,7 +75,7 @@ func Users(query string) (database.GetUsersParams, []codersdk.ValidationError) {
 	return filter, parser.Errors
 }
 
-func Workspaces(query string, page codersdk.Pagination, agentInactiveDisconnectTimeout time.Duration) (database.GetWorkspacesParams, []codersdk.ValidationError) {
+func Workspaces(ctx context.Context, db database.Store, query string, page codersdk.Pagination, agentInactiveDisconnectTimeout time.Duration) (database.GetWorkspacesParams, []codersdk.ValidationError) {
```

Comment:
aside: I see why we do it in this package, but this is kind of a weird naming scheme for something that returns database query params.

Reply:
Fair. Maybe it should be named `WorkspacesFilter` or something. I won't address here though